op_crates/fetch/26_fetch.js

```diff
@@ -958,12 +1001,12 @@
     // @ts-expect-error because the use of super in this constructor is valid.
     constructor(input, init) {
       if (arguments.length < 1) {
-        throw TypeError("Not enough arguments");
+        throw new TypeError(
+          `Failed to construct "Request," requires at least 1 argument, received ${arguments.length}`,
+        );
```

Comment:
The old error here was absolutely terrible.

A specialization of one of the bootstrap utility `requiredArguments` for constructors would be nice.

Reply:
These utils are already available in window.__bootstrap.webidl.

Reply:
Ah, I see, updating to make use of them.

Reply:
Actually, seems like window.__bootstrap.webidl doesn't include anything for a WebIDL dictionary, or I'm not seeing it.

If there isn't, I can add the cast there.

Reply:
@lucacasonato Does Deno have conversions for any of the following: HeadersInit, BodyInit, ReferrerPolicy, RequestMode, RequestCredentials, RequestCache, RequestRedirect, or AbortSignal?

[RequestInit IDL definition](https://fetch.spec.whatwg.org/#requestinit)

Most of these definately aren't WebIDL primitives.

Currently, I've only got the following:
```js
const requestInitConverter = createDictionaryConverter("RequestInit", [
  {
    converter: webidl.converter.ByteString,
    key: "method",
  },
  {
    converter: (v) => new Headers(v),
    key: "headers",
  },
  {
    converter: webidl.converter.USVString,
    key: "referrer",
  },
  {
    converter: webidl.converter.DOMString,
    key: "integrity",
  },
  {
    converter: webidl.converter.boolean,
    key: "keepalive",
  },
]);
```